core/src/language/mod.rs

```diff
@@ -18,6 +19,39 @@ pub use scala::Scala;
 pub use swift::Swift;
 pub use typescript::TypeScript;
 
+/// All supported languages.
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
+pub enum SupportedLanguage {
+    Go,
+    Kotlin,
+    Scala,
+    Swift,
+    TypeScript,
+}
```

Comment:
We use `#[deny(missing_docs)]`, so these variants need documentation. They're self-explanatory though, so I'd have no problem using `#[allow(missing_docs)]` on this enum.

Reply:
Yeah, there isn't really too much to document here, so just went with your allow lint. done!